generators/server/templates/src/main/java/package/config/PageableSpringEncoder.java.ejs

```diff
@@ -0,0 +1,72 @@
+package <%=packageName%>.config;
+
+import feign.RequestTemplate;
+import feign.codec.EncodeException;
+import feign.codec.Encoder;
+import org.springframework.data.domain.Pageable;
+import org.springframework.data.domain.Sort;
+
+import java.lang.reflect.Type;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+
+/**
+ * Provides support for encoding spring Pageable via composition.
```

Comment:
I'm not sure we can include this class from "Pascal Bütteriker" as part of the genrerated code. The license of this file is not clear for me. Ideally, we need to ask this person, if he agrees that his code (I know it's only one file but still...) be included in the JHipster lib, otherwise it will be "UNLICENSED" as part of the generated jhipster code and Mr Bütteriker might not agree to that.

Reply:
Sure, will try to find and ask him. The original code came from a PR to spring feign, so I guessed it was aclv2...